### ISSUE TITLE
Split validator page into three tables (with new / different data columns)

### ISSUE THREAD

**Issue by ckpringle** [vegaprotocol/frontend-monorepo#1263]
## Chore

Update validator page to separate consensus standby and candidate validators, updating the data columns to make it easier to decide who to stake and reduce the "how to" component visually to emphasise the list of validators.

The data columns have been updated (accurate in the sketch below) and the API field names used are in this spreadsheet  - https://docs.google.com/spreadsheets/d/1V28JXvQg2lC3-syayrnN5ysno8keUdg0YMfTtFQqu54/edit#gid=174802925

1. When you use fields from the rankingScore section they need to be from the current epoch
2. When you use fields from the rewardScore section of the API they need to be from prior epoch

## Sketch

![Image](https://user-images.githubusercontent.com/46783148/203095614-98527910-c7b0-4dc5-a080-2ff6b72ef49d.png)

https://miro.com/app/board/uXjVPUTU6m4=/?moveToWidget=3458764538296810799&cot=14

## Calculations 

**How to calculate 'required stake' for Standby banner**
Required = (rankingScore of the lowest ranked consensus validator) x total stake (sum of stake from all consensus validators)

Example 

- Lowest ranking score of 0.04488798832
- Total stake 32,652,108
- Required = 1465687.454

**How to calculate 'stake needed for promotion'...**

- We generated a required stake in the Standby banner which is the stake required assuming perfect performance right?  That is "Required"
- To calculate stake needed for promotion .....
-- Take that amount, scaled for that standby validators ACTUAL performance (hence the note about assuming constant performance) minus their current stake: Stake needed = Required / performance - stake
-- The equation in the spread sheet was "MAX(0, (Required / Performance) - Stake)" - the max part is just because otherwise we'd get minus numbers for any standby ones that had reached the level they needed to and were about to get promoted, so that makes sure in that case you get a zero out.
- Example
-- So taking "Candida"
-- Required Stake = 1,465,687 as per the banner
-- Stake Needed = 1,465,687 / 1 - 535,699 = 929,988

## Tasks 
- [ ] Design decision on whether we really need the same button over and over 
- [ ] Implement
- [ ] Test
- [ ] etc.

**Comment by JonRay15** [comment]
So ... where I think we eventually got to on this from a slack history check...

**Columns we want on the page:**
- Status (Needs to include Consensus, Standby (Ersatz), and Candidate)
- Total stake
- Self-stake
- Share
- Change next epoch
- Performance (from performance score)
- Reward score (from Normalised reward score)

Everything else can be removed 

Other requirements
- List should be sorted based on total stake with the largest at the top 
- List should by default hide validators that command the top 50% of the total staked on the network
- Users should have the option to expand that hidden section if desired, with some sort of dialog indicating to increase decentralisation they should not stake to these (see https://solanabeach.io/validators for the pattern we're aiming to follow here).
- Hover overs for each of the column headers explaining what the columns are showing need to be added
- Ideally given I've cut it down to only 7 columns we could get it all shrunk to fit the screen and remove the scroll

Questions: 
1. Is there any way we can show penalties?
2. Is there any way we could show an APY? 
3. Or if not can we show a "bang for your buck" metric?  ie. normalised reward score takes into account stake so it is higher for the ones with most stake.  Ideally we could normalise stake out of this so you can see how "efficient" staking with that validator will be.  Not sure what that would be..

**Comment by JonRay15** [comment]
Made a few more tweaks to this to simplify it.  Think its good to go now, subject to approval from the relevant lead.

**Comment by JonRay15** [comment]
In terms of priority .... we aim to expand the validator set in the weeks after launch. So this does not need to be in place for 17th Oct and the final release to validators, but needs to be in place by latest 3rd Nov ahead of that process starting.

**Comment by dexturr** [comment]
How do we calculate stake share?

**Comment by dexturr** [comment]
I think we should add a task to post an information message to sentry when pageInfo tells us we have more than a single page of nodes, if we are going to be doing client side filtering so we don't lose visibility of this.

**Comment by ckpringle** [comment]
@joellecharlotte made a start on the design of this but we didn't go too far (thinking we'll need to collab re: AG grid) 

![Image](https://user-images.githubusercontent.com/46783148/196374285-0178eb14-625a-4521-a688-34c336850e90.png)

https://www.figma.com/file/X9l0t8TZUF1tU8u0ILV3ws/Token-Frontend?node-id=1530%3A1837

**Comment by JonRay15** [comment]
FYI you can probably hold fire on this for a little while .... based on discussion between me, Candida, Barney and Jeremy today we've concluded we're not sure anyone fully understands all this so we want to do some analysis to work through all these figures, then agree which of those we want, and potentially request they be added to the API if not there already. 

Until thats done I would put this ticket on hold - except for general visual presentation stuff ... you can keep doing that if you like, we will just be changing the actual columns / data presented.

**Comment by sam-keen** [comment]
@ckpringle @dexturr one to be re-prioritised (see comment above)

**Comment by JonRay15** [comment]
I have now got a working prototype of what we want to see in this [spreadsheet](https://docs.google.com/spreadsheets/d/1V28JXvQg2lC3-syayrnN5ysno8keUdg0YMfTtFQqu54/edit#gid=174802925).

It should in theory be self-explanatory because the two rows labelled API / Calculation gives full details of where to grab the data and any calculations. For avoidance of doubt I used the graphQL below to grab all these necessary fields. 

Some other high level requirements to be aware of:

1. When you use fields from the rankingScore section they need to be from the current epoch 
2. When you use fields from the rewardScore section of the API they need to be from prior epoch

There is some scope for you to work out how best to present this but we're leaning towards:
1. Have seperate tables for Consensus validators and Standby/Pending validators 
2. This allows us to have different columns since Voting Power isnt relevant for standby, and Stake Needed For Promotion clearly isnt relevant for Consensus validators
3. Have some sort of banner in between that highlights how you move up; I've greyed all this out because I used to have a banner with the required stake x performance in it, but actually I wonder if this can just be text, and it could include the footnote there too ... also some of the text there is explanatory
4. The crucial bit is the extra detail shown off to the right.  We need this included but in the least visually invasive way possible .... suggestions included collapsing columns (most relate to total penalties, one relates to voting power so ideally arrange thus), doing them in a different colour off to the right, and including them as smaller less white text in the same cell....I leave it to you to work out what you like best

`{
  epoch (id:960) {
    id
    validatorsConnection{
      edges{
        node {
          id
          stakedTotal
          rewardScore{
            rawValidatorScore
            performanceScore
            validatorScore
            normalisedScore
            multisigScore
            validatorStatus
          }
          rankingScore{
            stakeScore
            performanceScore
            previousStatus
            status
            votingPower
            rankingScore
          }
        }
      }
    }
  }
}
`
